lib/jekyll/converters/markdown/kramdown_parser.rb

```diff
@@ -5,19 +5,20 @@ module Jekyll
   module Converters
     class Markdown
       class KramdownParser
+        attr_accessor :config
         CODERAY_DEFAULTS = {
-          "css"               => "style",
-          "bold_every"        => 10,
-          "line_numbers"      => "inline",
+          "css" => "style",
+          "bold_every" => 10,
+          "line_numbers" => "inline",
```

Comment:
I was trying to keep the hash rockets all aligned because I think it reads more easily like a table than like this.